torchao/quantization/prototype/qat/embedding.py

```diff
@@ -98,28 +160,31 @@ def _convert_helper(self, module: torch.nn.Module):
         from torchao._executorch_ops import _quantized_decomposed_quantize_per_channel_group_wrapper
         for name, child in module.named_children():
             if isinstance(child, Int4WeightOnlyQATEmbedding):
+                group_size = child.weight_fake_quantizer.config.group_size
+                scale_precision = child.weight_fake_quantizer.config.scale_precision
+                zero_point_precision = child.weight_fake_quantizer.config.zero_point_precision
                 quantized_embedding = Int4WeightOnlyEmbedding(
-                    group_size=child.group_size,
-                    scale_precision=child.scale_precision,
-                    zero_point_precision=child.zero_point_precision,
-
-                    # other nn.Embedding args
+                    # nn.Embedding args
                     num_embeddings=child.num_embeddings,
                     embedding_dim=child.embedding_dim,
                     padding_idx=child.padding_idx,
                     max_norm=child.max_norm,
                     norm_type=child.norm_type,
                     scale_grad_by_freq=child.scale_grad_by_freq,
                     sparse=child.sparse,
+                    # quantization args
+                    group_size=group_size,
+                    scale_precision=scale_precision,
+                    zero_point_precision=zero_point_precision,
                     device=child.weight.device,
                 )
                 setattr(module, name, quantized_embedding)
 
                 # Load weights and qparams into quantized embedding
-                (qmin, qmax) = _get_qmin_qmax(self.bit_width)
-                (s, zp) = get_group_qparams_symmetric(child.weight, self.bit_width, child.group_size)
+                (qmin, qmax) = _get_qmin_qmax(4)
```

Comment:
I feel it might be better to keep `4` as an constant defined in the class so we can generalize later and we'll know more semantic information here for the arg